lib/cli_interface.rb

```diff
@@ -26,20 +24,34 @@ def set_lang(input)
     @text_file = YAML::load(File.open(file))
   end
 
-  def menu_game
-    write("#{get_from_file('menu_type_games')}")
+  def display_menu_type_game(default_checked_type = 1)
+    display_menu(get_from_file('menu_type_games'), default_checked_type)
   end
 
-  def menu_lang
-    display_lang_files
+  def menu_type_game_size
```

Comment:
It's the second case. I agree, that's really fragile. May be I should put a char at the start of each line and count the lines which start_with?